.hound.yml

```diff
@@ -8,6 +8,7 @@ jshint:
   enabled: false
 
 haml:
+  enabled: false

```

Comment:
I removed the HAML lint step because Hound does it as well, which is better for the review/PR approach.

But haml-lint infers RuboCop cops (locally) and that makes sense for stuff like quote rules and such, and Hound can't resolve these either, so either we go for a single-quote style there and disable Cop stuff in HAML, or we just don't lint with Hound here as well.